torch/ao/quantization/pt2e/qat_utils.py

```diff
@@ -705,20 +738,24 @@ def _fold_conv_bn_qat_helper(m: GraphModule, is_cuda: bool) -> GraphModule:
         [True, False],  # has_relu
         [True, False],  # has_bias
         [True, False],  # relu_is_inplace
+        [True, False],  # has_add
+        [True, False],  # add_is_inplace
```

Comment:
Just saw this. Just wanted to point out that this exponentially increases the number of patterns we need to match against. Right now we have 6 options here + 1 for cuda vs not cuda. That's already 128 patterns. I'm adding quantized bias support here (https://github.com/pytorch/pytorch/pull/112528) which adds another dimension, bringing this up to 256 patterns. This seems concerning to me.

Reply:
Probably fine for these two PRs but I think we should really think about how to push some of this functionality into the subgraph rewriter so we don't have hundreds of patterns to match. cc @jerryzh168